<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigSystem</class>
 <widget class="QWidget" name="frmConfigSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="1" column="1">
       <widget class="QGroupBox" name="gboxColor">
        <property name="title">
         <string>颜色设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="1">
          <widget class="ColorComboBox" name="cboxColorZero">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labColorZero">
           <property name="text">
            <string>离线颜色</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labColorPlotBg">
           <property name="text">
            <string>曲线背景</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="ColorComboBox" name="cboxColorPlotBg">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labColorUpper">
           <property name="text">
            <string>高报颜色</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labColorPlotText">
           <property name="text">
            <string>曲线文字</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ColorComboBox" name="cboxColorUpper">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labColorLimit">
           <property name="text">
            <string>低报颜色</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ColorComboBox" name="cboxColorLimit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labColorPlotLine">
           <property name="text">
            <string>曲线颜色</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="ColorComboBox" name="cboxColorPlotText">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="ColorComboBox" name="cboxColorPlotLine">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="labColorOther">
           <property name="text">
            <string>其他颜色</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labColorNormal">
           <property name="text">
            <string>正常颜色</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="ColorComboBox" name="cboxColorNormal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="ColorComboBox" name="cboxColorOther">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="gboxBase">
        <property name="title">
         <string>常规设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="2">
          <widget class="QLabel" name="labStyleName">
           <property name="text">
            <string>界面样式</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="3">
          <widget class="QLineEdit" name="txtAuthor">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labAuthor">
           <property name="text">
            <string>版权所有</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="SwitchButton" name="btnAutoConfirm" native="true"/>
         </item>
         <item row="2" column="1">
          <widget class="SwitchButton" name="btnAutoPwd" native="true"/>
         </item>
         <item row="0" column="1">
          <widget class="SwitchButton" name="btnAutoRun" native="true"/>
         </item>
         <item row="1" column="1">
          <widget class="SwitchButton" name="btnAutoLogin" native="true"/>
         </item>
         <item row="3" column="3">
          <widget class="QComboBox" name="cboxStyleName"/>
         </item>
         <item row="3" column="5">
          <widget class="QComboBox" name="cboxWorkMode"/>
         </item>
         <item row="1" column="3" colspan="3">
          <widget class="QLineEdit" name="txtLogoEn"/>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="labWorkMode">
           <property name="text">
            <string>工作模式</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labLogoC">
           <property name="text">
            <string>中文标题</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labLogoE">
           <property name="text">
            <string>英文标题</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" colspan="3">
          <widget class="QLineEdit" name="txtLogoCn"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labAutoLogin">
           <property name="text">
            <string>自动登录</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labAutoPwd">
           <property name="text">
            <string>记住密码</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labAutoRun">
           <property name="text">
            <string>开机运行</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labAutoConfirm">
           <property name="text">
            <string>自动确认</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="gboxTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>系统时间设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QComboBox" name="cboxHour">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labHour">
           <property name="text">
            <string>时</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="cboxMin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labMin">
           <property name="text">
            <string>分</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QComboBox" name="cboxSec">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="labSec">
           <property name="text">
            <string>秒</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6" rowspan="2">
          <widget class="QPushButton" name="btnSetTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>设置</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="cboxDay">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="labDay">
           <property name="text">
            <string>日</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="cboxYear">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="cboxMonth">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labYear">
           <property name="text">
            <string>年</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="labMonth">
           <property name="text">
            <string>月</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>switchbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>colorcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtLogoCn</tabstop>
  <tabstop>txtLogoEn</tabstop>
  <tabstop>txtAuthor</tabstop>
  <tabstop>cboxStyleName</tabstop>
  <tabstop>cboxColorZero</tabstop>
  <tabstop>cboxColorPlotBg</tabstop>
  <tabstop>cboxColorUpper</tabstop>
  <tabstop>cboxColorPlotText</tabstop>
  <tabstop>cboxColorLimit</tabstop>
  <tabstop>cboxColorPlotLine</tabstop>
  <tabstop>cboxColorNormal</tabstop>
  <tabstop>cboxColorOther</tabstop>
  <tabstop>cboxYear</tabstop>
  <tabstop>cboxMonth</tabstop>
  <tabstop>cboxDay</tabstop>
  <tabstop>cboxHour</tabstop>
  <tabstop>cboxMin</tabstop>
  <tabstop>cboxSec</tabstop>
  <tabstop>btnSetTime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
